analyzer.go

```diff
@@ -185,8 +191,8 @@ func readConfig(result *styleConfig, path string) {
 func skip(funcNode ast.Node) bool {
 	switch funcNode := funcNode.(type) {
 	case *ast.FuncDecl:
-		if strings.HasPrefix(funcNode.Name.Name, "_Cfunc_") {
-			return true
+		if strings.HasPrefix(funcNode.Name.Name, "_Cfunc_") { // I don't think it is safe to assume Cgo functions always start with _Cfunc_
+			return !config.IncludeCgo
 		}
```

Comment:
```suggestion
		if strings.HasPrefix(funcNode.Name.Name, "_Cfunc_") && !config.IncludeCgo {
			return true
		}
```

Reply:
This way we don't return early if this is cgo function, and allow other checks in this function (if we'll add them later) to be tested too.